src/components/molecules/card/card.scss

```
@@ -0,0 +1,52 @@
+@use "../../variables" as *;
+
+.card {
+  border-radius: 0;
+  overflow: hidden;
+  border: none;
+
+  .card-top {
+    position: relative;
+    overflow: hidden;
+
+    .card-title-triangle {
+      position: absolute;
+      border-top: 150px solid $white;
+      border-right: 150px solid transparent;
+      z-index: 1;
+    }
+    .card-title-text {
+      position: absolute;
+      font-size: $fontsize-l;
+      font-weight: 700;
+      top: 0;
+      left: 4px;
+      line-height: 29px;
+      color: $orange-bistre;
+      z-index: 2;
+    }
+    .card-image {
+      width: 100%;
+      transition: all 0.5s ease;
```

```suggestion
      transition: transform 0.5s ease;
```
このようにするとtransform以外に適用されることがなくなるので安心です！
意図しない副作用を防ぐという考え方を持っておくといいと思います 👏 
現状でも問題はないので修正はお好みでお願いします。